packages/tracing-internal/src/browser/browsertracing.ts

```diff
@@ -327,8 +327,14 @@ export class BrowserTracing implements Integration {
     let inflightInteractionTransaction: IdleTransaction | undefined;
     const registerInteractionTransaction = (): void => {
       const { idleTimeout, finalTimeout, heartbeatInterval } = this.options;
-
       const op = 'ui.action.click';
+
+      const currentTransaction = getActiveTransaction();
+      if (currentTransaction?.op && ['navigation', 'pageload'].includes(currentTransaction.op)) {
```

Comment:
```suggestion
      if (currentTransaction && currentTransaction.op && ['navigation', 'pageload'].includes(currentTransaction.op)) {
```

We should avoid optional chaining, as that is transpiled to very verbose code.

Reply:
@mydea We should add to the docs and / or include a lint rule to prevent optional chaining, so newer contributors don't make the same mistake! 😄

Reply:
hmm we actually have a lint rule for this 😅  I see we have manually disabled this in the eslintrc for tracing-internal, we should remove this there!